node/src/tls.rs

```diff
@@ -81,7 +81,7 @@ type SslResult<T> = Result<T, ErrorStack>;
 
 /// SHA512 hash.
 #[derive(Copy, Clone, DataSize, Deserialize, Serialize)]
-struct Sha512(#[serde(with = "big_array::BigArray")] [u8; Sha512::SIZE]);
+pub struct Sha512(#[serde(with = "big_array::BigArray")] [u8; Sha512::SIZE]);
```

Comment:
Does inlining of the `NodeId` require this struct to be `pub` now?

Reply:
Doesn't require it, but the PR did that IIRC for something related to testing.

Reply:
It may actually be unnecessary now, but I shouldn't make that change on this merge PR.